source/site-administrators/studio/rte2-setup.rst

```diff
@@ -73,6 +73,40 @@ The default editor instance contains a menubar with most of the commonly used ed
 
 |
 
+.. _rte-paste-plugin-hooks:
+
+TinyMCE paste plugin callback hooks
```

Comment:
The `craftercms_tinymce_hooks` is sort of general purpose. It's not just for paste. Currently, it allows for hooking into paste pre/post process **and** Tiny's setup function (as shown by the example). People may use either/or as as needed. We should mention the `craftercms_tinymce_hooks` as a general tool to extend tiny and list it's current usages somewhere in the docs. As that evolves with possibly new hooks, that'll need to be maintained.